srcpkgs/66-boot/INSTALL

```diff
@@ -0,0 +1,10 @@
+if [ "$UPDATE" = "no" ]; then
+	case "${ACTION}" in
+	post)
+		66-tree -n boot
+		66-enable -t boot boot@system
+		66-tree -ncE default
+		66-enable -t default tty@tty1 tty@tty2 tty@tty3 tty@tty4
```

Comment:
Not sure if its a good idea to do all of this in INSTALL, also wouldn't work right when doing install with `xbps-install --root`.

Reply:
I will create a doc for docs.voidlinux.org